integration

```diff
@@ -212,6 +212,7 @@
   "davesmeghead/visonic",
   "dckiller51/bodymiscale",
   "dcmeglio/homeassistant-petsafe",
+  "dcmeglio/homeassistant-waste_management",
```

Comment:
This is impossible to use.
There are nothing that show which service/which country
It requires user/pass, but without knowing to what it just cant be used by anyone (other than the author)

Reply:
I believe you're thinking this is some generic name, but Waste Management is the name of the company. It's the largest garbage removal company in the world serving most of North America. The integration includes their logo too.  They have 30 million customers so I think it's useful to others aside from me, https://www.wm.com/

Reply:
A link should be added as a minimum.
I did a google search for “waste management” and got multiple hits for every country in the world

Reply:
And it needs a list of countries in hacs.json

Reply:
I updated the repo description to "Determines your next Waste Management (https://www.wm.com/) pickup times for use in Home Assistant." so hopefully that avoids confusion. Added the countries listed on their website to the hacs.json, https://github.com/dcmeglio/homeassistant-waste_management/blob/master/hacs.json

Latest HACS run, https://github.com/dcmeglio/homeassistant-waste_management/actions/runs/3920462676
Latest hassfest run, https://github.com/dcmeglio/homeassistant-waste_management/actions/runs/3920462674

Let me know if anything else is needed